samples/sample-generic-content-sidekick-plugin/src/components/sample-generic-content-sidekick-plugin-item/component.tsx

```diff
@@ -53,6 +59,23 @@ function SampleGenericContentSidekickPlugin(
     ]);
   }, []);
 
+  useEffect(() => {
+    pluginApi.setActionButtonDropdownItems([
+      new ActionButtonDropdownOption({
+        label: 'Click to increment the badge',
+        icon: 'user',
+        tooltip: 'Use it to enable the badge',
+        allowed: true,
+        onClick: () => {
+          setCountGenericContent((curr) => curr + 1);
+          pluginApi.uiCommands.sidekickArea.options.setMenuBadge(
+            GENERIC_CONTENT_BADGE_ID,
+            countGenericContent.toString(),
+          );
+        },
+      }),
+    ]);
+  }, [countGenericContent]);
```

Comment:
It would be good to have an example of use for each new sidekick area option:
```suggestion
      new ActionButtonDropdownOption({
        label: 'Click to change section name',
        icon: 'plus',
        tooltip: 'Use it to change section name',
        allowed: true,
        onClick: () => {
          pluginApi.uiCommands.sidekickArea.options.renameGenericContentSection(
            GENERIC_CONTENT_BADGE_ID,
            `New Section Name ${countGenericContent.current.toString()}`,
          );
        },
      }),
      new ActionButtonDropdownOption({
        label: 'Click to change menu name',
        icon: 'edit',
        tooltip: 'Use it to change menu name',
        allowed: true,
        onClick: () => {
          pluginApi.uiCommands.sidekickArea.options.renameGenericContentMenu(
            GENERIC_CONTENT_BADGE_ID,
            `New Menu Name ${countGenericContent.current.toString()}`,
          );
        },
      }),
  }, []);
```